components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/GatewayArtifactsMgtDAO.java

```diff
@@ -517,6 +513,79 @@ public List<APIRuntimeArtifactDto> retrieveGatewayArtifactsByAPIIDs(List<String>
         return apiRuntimeArtifactDtoList;
     }
 
+    public List<APIRuntimeArtifactDto> retrieveGatewayArtifactsOnlyByAPIIDs(List<String> apiIds, String tenantDomain)
+            throws APIManagementException {
+        // Logging the API ID List
+        log.debug("Getting runtime artifacts for the API ID List: " + apiIds.toString());
```

Comment:
Check whether debug is enabled here.

Reply:
Fixed: https://github.com/wso2/carbon-apimgt/pull/11304/commits/efe9a404f15c7e0d4bf3be5ec538ec30091b0f03